avocado/utils/linux_modules.py

```diff
@@ -40,7 +40,12 @@
 
 
 def load_module(module_name):
-    # Checks if a module has already been loaded
+    """
+    Checks if a module has already been loaded.
+    :param module_name: Name of module to check
+    :return: True if module is loaded, False otherwise
+    :rtype: Bool
+    """
     if module_is_loaded(module_name):
         return False
```

Comment:
@apahim i think changing name of function can broke  external test cases but i think @lmr suggested a RFC #1686 so we can change what you say  @ldoktor any input ?